deploy/mobileclient_admin_cluster_role.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: mobileclient-admin-cluster
+rules:
+  - apiGroups:
+      - mobile.k8s.io
+    resources:
+      - mobileclients
```

Comment:
As far as I can tell, we didn't have to add such a ClusterRole[Binding] for our other CRDs. Do we have any idea why it's needed for mobileclients? i.e. why do we see that error in the web console for that kind, and not for, say, PushApplication?